compiler/rustc_typeck/src/check/expr.rs

```diff
@@ -1348,29 +1348,44 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         span: Span,
         remaining_fields: FxHashMap<Ident, (usize, &ty::FieldDef)>,
     ) {
-        let tcx = self.tcx;
-        let len = remaining_fields.len();
+        let mut len = remaining_fields.len();
```

Comment:
`mut` variables always raise my alert level a little to see where any why they change. Having `len - 3` twice below is fine if you ask me, but if you prefer having it just once, you could do `let len = len - 3` instead of `len -=3`.
No super storing feelings though.

Reply:
I changed it.